vertexai/chat/chat_test.go

```diff
@@ -15,15 +15,19 @@
 package chat
 
 import (
+	"context"
+	"os"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
 func Test_makeChatRequests(t *testing.T) {
-	t.Skip("TODO(muncus): remove skip")
+	if os.Getenv("GOLANG_SAMPLES_PROJECT_ID") == "" {
```

Comment:
This check is already performed in `testutil.SystemTest` below, and is redundant here. please remove.

Reply:
Done.